aiida_wannier90/__init__.py

```diff
@@ -16,9 +20,9 @@
 ## If upgraded, remember to change it also in setup.json (for pip)
 __version__ = "2.0.0a1"
 
-from . import io
-from . import parsers
-from . import orbitals
+from . import io  # noqa: disable=F401
+from . import parsers  # noqa: disable=F401
+from . import orbitals  # noqa: disable=F401
 ## If this is imported then AiiDA ORM is loaded and this fails if the load_dbenv has not
```

Comment:
I think this comment is outdated -- previously `calculations` could not be imported without a loaded dbenv, but it appears this is no longer the case (there were some changes in `aiida-core` affecting this).

Reply:
Great!